src/react/components/edgeScroll.ts

```
@@ -0,0 +1,104 @@
+      (overflowY === "auto" || overflowY === "scroll") &&
+      el.scrollHeight > el.clientHeight;
+    if (scrollable) return el;
+    el = el.parentElement;
+  }
+  return null;
+}
+
+export function createEdgeScrollController(
+  node: HTMLElement | null
+): EdgeScrollController {
+  let scrollContainer: HTMLElement | null = getNearestScrollable(node);
```

ESLint

[error] 'scrollContainer' is never reassigned. Use 'const' instead.

(prefer-const)